packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItem.types.ts

```diff
@@ -19,8 +19,9 @@ export interface TreeItemProps {
   item: GQLTreeItem;
   itemIndex: number;
   depth: number;
-  onExpand: (id: string, depth: number) => void;
-  onExpandAll: (treeItem: GQLTreeItem) => void;
+  expanded: string[];
+  maxDepth: number;
+  onExpandedElementChange: (expanded: string[], maxDepth: number) => void;
```

Comment:
We will probably have to extract this concern into a dedicated hook later

Reply:
Ok, I didn't do anything in this regard in this PR.